### PR TITLE
Fix embedded documents 

### PR DESCRIPTION
**Pull request by minitriga**
Related Issue: #253 

## New Behavior
Shows embedded documentation in Netbox. 

## Contrast to Current Behavior
Docs were excluded from image build. 

## Double Check

* [x] I have read the comments and followed the PR template.
* [x] I have explained my PR according to the information in the comments.
* [x] My PR targets the `develop` branch.

### CODE REVIEW

**Comment by tobiasge** [comment]
The change looks good. I tested the same thing locally.

But there seems to be a commit in this PR that doesn't belong. Could you fix this?

**Comment by minitriga** [comment]
> The change looks good. I tested the same thing locally.
> 
> But there seems to be a commit in this PR that doesn't belong. Could you fix this?

Looks like I messed it up, even more, let me try again. Not the best with git at times.
